### DISCUSSION TITLE
Extend support for XHamster

### DISCUSSION THREAD

**Discussion by BlueJay-66** [AAndyProgram/SCrawler#118] [Ideas]
Hi!

First of all I'd like to say "thank you" for your awesome work so far. Much appreciated!

Now here's my suggestion/request: 
Until now, XHamster posts have to be like https://xhamster.com/users/<user name>. Would it be possible to also allow https://xhamster.com/creators/<creator name> or https://xhamster.com/channels/<channel name>?

Thanks for considering.

Regards
BlueJay-66

**Comment by AAndyProgram** [comment]
Duplicate of #107 
Channels were added in release 2023.3.1.0.
Creators: https://github.com/AAndyProgram/SCrawler/issues/107#issuecomment-1436184558
